resources/lang/en/people.php

```diff
@@ -276,5 +276,7 @@
     'debt_add_amount' => 'the sum of',
     'debt_add_reason' => 'for the following reason (optional)',
     'debt_add_add_cta' => 'Add debt',
+    'debt_edit_update_cta' => 'Update debt',
```

Comment:
I'm going to be super annoying, but you've introduced new keys, which is not a problem - but they new keys should also be put on the other translation files as well (you don't need to translate them, just paste them as is).